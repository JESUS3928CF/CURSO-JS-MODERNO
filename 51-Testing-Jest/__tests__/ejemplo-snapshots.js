/// Los snapshots son datos que se almacenan en uin string se crean en una carpeta aparte y sobre ella podemos comparar todo el objeto

const cliente = {
    nombre: 'Jesús 2',
    balance: 600,
    tipo: "Premium"
};


describe("Testing al cliente", () => {
    test('Es Jesús Cochero', () => {
        //. Si lo ejecutamos esto crear una carpeta de snapshots
        //* Ese archivo lo administra JEST osea tu no
        expect(cliente).toMatchSnapshot('Jesús Cochero'); /// Método Igual a
    });
});