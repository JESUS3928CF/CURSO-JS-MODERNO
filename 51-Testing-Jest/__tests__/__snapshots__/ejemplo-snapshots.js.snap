// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Jesús Cochero: Jesús Cochero 1`] = `
{
  "balance": 600,
  "nombre": "Jesús 2",
  "tipo": "Premium",
}
`;
